exercises/.byebug_history

```diff
@@ -0,0 +1,18 @@
+exit
```

Comment:
That can be deleted, it was automatically created by a debugging gem. Thanks!

Sent from mobile

> On May 14, 2016, at 12:07 PM, Katrina Owen <notifications@github.com> wrote:
> 
> In exercises/.byebug_history:
> 
> > @@ -0,0 +1,18 @@
> > +exit
> What does this file do?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>